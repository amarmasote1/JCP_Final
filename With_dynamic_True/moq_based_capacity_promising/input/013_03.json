[
  {
    "storeID": "2456",
    "itemID": "54003001012",
    "uom": "EACH",
    "supplyData": {
      "onhandsupply": 20
    },
    "demandData": {
        "reserveddemand": 0
      },
    "timestamp": "<$currentTime>"
    
  }
]