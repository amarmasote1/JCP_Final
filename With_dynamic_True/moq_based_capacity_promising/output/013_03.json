{
    "message": "Success",
    "body": [
        {
            "itemID": "54003001012",
            "uom": "EACH",
            "demandData": {
                "reserveddemand": 0
            },
            "markDown": {
                "active": false
            },
            "storeID": "2456",
            "supplyData": {
                "onhandsupply": 20
            },
            "timestamp": "2020-02-28 12:45:54.0000 IST"
        }
    ],
    "status": 200
}