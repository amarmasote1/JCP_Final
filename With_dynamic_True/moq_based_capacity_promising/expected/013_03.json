{
	"status": 200,
	"message": "Success",
	"body": [{
			"itemID": "54003001012",
			"uom": "EACH",
			"storeID": "2456",
			"supplyData": {
				"onhandsupply": 20
			},
			"demandData": {
				"reserveddemand": 0
			}
		}
	]
}
